### DISCUSSION TITLE
Bluetooth button rewinds but doesn't go to previous track

### DISCUSSION THREAD

**Discussion by MagTun** [PaulWoitaschek/Voice#1890] [Ideas]
**Is your feature request related to a problem? Please describe.**
Bluetooth button is able to rewind but only on the current track. Once I reached the start of the current track, the Bluetooth rewinds doesn't work anymore. When I need to to to the end of the previous track, I have to use the button the widget, I can't use my Bluetooth headphone.

**Describe the solution you'd like**
Same behavior on the Bluetooth rewind than on the widget rewind: ability to jump to previous track, once the sart of the current track has been reach

Voice version 5.0.1

**Comment by jtagcat** [comment]
Duplicate of #866

**Comment by MagTun** [comment]
@jtagcat, this is not a duplicate of #866. #866 wants the buttons to forward/backward one track (instead of forward/backward several seconds). I do not request this. 


What I want is :
- I am at 30 second on track 10
- I press several times the backward and Voice goes backward : Voice goes to track 10 at 20 sec, then track 10 at 10 sec, then track 10 at 0sec (assuming the skipping time is 10secs)
- I keep pressing the backward button: instead being stuck at track 0sec on track 10 (which is the current behaviors), Voice should jump to track 9 4m50s (assuming track 9 is 5min long) and if I keep pressing, track 9 4min40 and so on.  This "should" is my feature request.

So in brief: I want the button to forward/backward several seconds which is the current behaviors, but once the backward button reaches the beginning of,  let's say, track 10, then Voice should goes to the end of track 9

**Comment by MagTun** [comment]
I was able to rewind to the previous track but it's a bit tricky at first.
So let's say I want to go back 20 sec in my audiobook but I am 5sec into track 11. 
If I do a long press of about 2 sec, on the back button of my bluetooth headset, I will then hear the audio start at the beginning of track 11.
The trick I found was to press the back button a bit less than 2 sec (maybe 1.5sec : I stop to press just before I feel that the audio will start again), and repress immediately a second time for a long press of 2sec. Now when I release I am about 10 sec away from the end of track 10. 

But it would be nice not to have to rely and such unreliable method.
